### PR TITLE
[vtctldclient] vtctldclient generator

### PR DESCRIPTION
**Pull request by ajm188**
## Description

This adds a code generator to produce the boilerplate "if underlying connection is nil, return closed; otherwise, call underlying method" for every unary RPC in the vtctld client. It does not handle streaming RPCs (yet) because we do not need streaming RPCs (yet)

## Related Issue(s)

- #7237 

## Checklist
- [ ] Should this PR be backported? **No.**
- [ ] Tests were added or are not required **N/A**
- [ ] Documentation was added or is not required **N/A**

## Impacted Areas in Vitess
Components that this PR will affect:

- [ ]  Query Serving
- [ ]  VReplication
- [ ]  Cluster Management
- [ ]  Build

### CODE REVIEW

**Review by doeg — APPROVED** [review]
This is so cool!!! Magical, even. (In a good way!) Thank you for doing this! 

I pulled your branch and re-generated client_gen.go and it looks good/worked great. (The automatic VS Code integration is so nice, too!) 

My only suggestion would be that maybe we could/should add a Makefile target, but I leave that up to you. :)

**Comment by ajm188** [comment]
I was just thinking about how I should have added a Makefile target. Good call.

File: go/vt/vtctl/grpcvtctldclient/Makefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 The Vitess Authors.
```
Comment (doeg) [inline go/vt/vtctl/grpcvtctldclient/Makefile:1]: Oh... right, it's 2021 now. 😭 Haha. I guess this applies to all the files in this branch.

**Review by deepthi — APPROVED** [review]
Very cool indeed! :100:

**Pull request merged**